prow/cmd/plank/main.go

```diff
@@ -112,6 +115,16 @@ func main() {
 		logger.WithError(err).Fatal("Error creating plank controller.")
 	}
 
+	// Push metrics to the configured prometheus pushgateway endpoint.
+	pushGateway := configAgent.Config().PushGateway
+	if pushGateway.Endpoint != "" {
+		go metrics.PushMetrics("plank", pushGateway.Endpoint, pushGateway.Interval)
+	}
+	// serve prometheus metrics.
+	go serve()
```

Comment:
Why do we need to serve the metrics from the controller if we are already pushing them to the push gateway?

Reply:
The gateway is optional in case you run prow in a locked-down env. We don't use it downstream.

Reply:
In any case we should be able to access metrics from prow using both ways (pull, push).